.github/ISSUE_TEMPLATE/hosted-data-browser-bug-report.yml

```diff
@@ -0,0 +1,54 @@
+name: Hosted Data Browser Bug report
+description: Data Browser is not working as expected on https://cloud.prisma.io
+labels: "kind/bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us improve Prisma! 🙏 Please follow the sections in the template and provide as much information as possible about your problem.
+  - type: textarea
+    attributes:
+      label: Bug description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: Steps to reproduce the behavior
+      value: |
+        <!--
+        1. Go to '...'
+        2. Change '....'
+        3. Run '....'
+        4. See error
+        -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: A clear and concise description of what you expected to happen.
+  - type: textarea
+    attributes:
+      label: Prisma information
```

Comment:
"information" seems a bit vague here, but I'm unsure how this will show up in the end. Would "Schema" work here?

I see that for the Studio version, "Prisma information" makes a lot more sense, as you're asking for info more than the schema. You could leave this as-is too, up to you.